Makefile

```diff
@@ -56,15 +56,22 @@ lint: lint-dependencies lint-all
 ############################################################
 # test section
 ############################################################
+
+ENVTEST = $(LOCAL_BIN)/setup-envtest
```

Comment:
Could you please set these configuration?
https://sdk.operatorframework.io/docs/upgrading-sdk-version/v1.7.1/#gov3-set-the-makefiles-shell-variable-to-the-systems-bash-binary